### ISSUE TITLE
How do i import SASS from node_modules more easily? (E.g. tilde character)

### ISSUE THREAD

**Issue by FunctionDJ** [glenn2223/vscode-live-sass-compiler#169]
With Next.js and the `sass` package installed, i can write `@import "bulmaswatch/cyborg/variables";` and it knows to import this file from the `bulmaswatch` package inside `node_modules`. This doesn't work when i run this extension:
```
Error: File "c:\Users\Function\Documents\Git\pplus-website\src\bulmaswatch\cyborg\variables" not found
        on line 1 of sass/c:\Users\Function\Documents\Git\pplus-website\src\custom-bulma.scss
>> @import "bulmaswatch/cyborg/variables"; // swatch styles
   --------^
```
I have to write `../node_modules/bulmaswatch/cyborg/variables` which is a bit annoying and also not consistent between different locations in my project.
The tilde character `~` that indicated the node_modules directory in certain SASS setups also doesn't work.

How can i make this better? Am i missing some configuration?

**Comment by glenn2223** [comment]
It's not currently an option, but it shouldn't be a problem to add the feature though (using a custom importer). Bear with me while I draft something up, I'll come back with an alpha for you to confirm the feature is working as you'd expect

**Comment by glenn2223** [comment]
As promised, install this and see if it works as you'd expect (you need to prefix with "~")

[live-sass-5.4.0-alpha.1.zip](https://github.com/glenn2223/vscode-live-sass-compiler/files/8419046/live-sass-5.4.0-alpha.1.zip)

**Comment by FunctionDJ** [comment]
@glenn2223 Just tried it out, and it works great!

**Comment by glenn2223** [comment]
Glad it works! I will push it officially soon

**Comment by glenn2223** [comment]
Sorry for the delay. Available in v5.4.0
